src/Raven.Server/Commercial/LicenseManager.cs

```diff
@@ -1548,22 +1548,23 @@ public bool HasHighlyAvailableTasks()
         public static AlertRaised CreateHighlyAvailableTasksAlert(DatabaseTopology databaseTopology, IDatabaseTask databaseTask, string lastResponsibleNode)
         {
             var taskName = databaseTask.GetTaskName();
-            var message = $"Node {lastResponsibleNode} cannot execute the task: '{taskName}'";
+            var taskType = GetTaskType(databaseTask);
+            var nodeState = GetNodeState(databaseTopology, lastResponsibleNode);
+            var message = $"Cannot redistribute the {taskType} task: '{taskName}'";
             var alert = AlertRaised.Create(
                 null,
-                $@"You've reached your license limit ({EnumHelper.GetDescription(LimitType.HighlyAvailableTasks)})",
+                $@"You've reached a license limit ({EnumHelper.GetDescription(LimitType.HighlyAvailableTasks)})",
                 message,
                 AlertType.LicenseManager_HighlyAvailableTasks,
                 NotificationSeverity.Warning,
                 key: message,
                 details: new MessageDetails
                 {
-                    Message = $"The {GetTaskType(databaseTask)} task: '{taskName}' will not be executed " +
-                              $"by node {lastResponsibleNode} (because it is {GetNodeState(databaseTopology, lastResponsibleNode)}) " +
-                              $"or by any other node because your current license " +
-                              $"doesn't include highly available tasks feature. " + Environment.NewLine +
-                              $"You can choose a different mentor node that will execute the task " +
-                              $"(current mentor node state: {GetMentorNodeState(databaseTask, databaseTopology)}). " +
+                    Message = $"The {taskType} task: '{taskName}' will not be redistributed " + 
+                              $"to healthy node because current license doesn't include highly available tasks feature." + Environment.NewLine +
```

Comment:
```suggestion
                              $"to a healthy node because the current license doesn't include the highly available tasks feature." + Environment.NewLine +
```